### ISSUE TITLE
Allow specifying gas price, fee currency

### ISSUE THREAD

**Issue by macalinao** [celo-tools/celo-web-wallet#36]
#### Feature Description

I'd love to see being able to set gas price (i think min price is 0.1 GWei)

Also, being able to choose between CELO/cUSD etc would be nice

**Comment by jmrossy** [comment]
Yep this would be great addition for sure! It's being considered for the next milestone :)
